debug_toolbar/panels/staticfiles.py

```diff
@@ -138,17 +139,21 @@ def get_staticfiles_finders(self):
         """
         finders_mapping = OrderedDict()
         for finder in finders.get_finders():
-            for path, finder_storage in finder.list([]):
-                if getattr(finder_storage, "prefix", None):
-                    prefixed_path = join(finder_storage.prefix, path)
-                else:
-                    prefixed_path = path
-                finder_cls = finder.__class__
-                finder_path = ".".join([finder_cls.__module__, finder_cls.__name__])
-                real_path = finder_storage.path(path)
-                payload = (prefixed_path, real_path)
-                finders_mapping.setdefault(finder_path, []).append(payload)
-                self.num_found += 1
+            try:
+                for path, finder_storage in finder.list([]):
+                    if getattr(finder_storage, "prefix", None):
+                        prefixed_path = join(finder_storage.prefix, path)
+                    else:
+                        prefixed_path = path
+                    finder_cls = finder.__class__
+                    finder_path = ".".join([finder_cls.__module__, finder_cls.__name__])
+                    real_path = finder_storage.path(path)
+                    payload = (prefixed_path, real_path)
+                    finders_mapping.setdefault(finder_path, []).append(payload)
+                    self.num_found += 1
+            except OSError:
+                # This error should be captured and presented as a part of run_checks.
```

Comment:
I'm not sure whether this is an old comment which is not relevant anymore or whether there's anything left to do?